website/azurerm.erb

```diff
@@ -857,6 +861,10 @@
                   <a href="/docs/providers/azurerm/r/eventgrid_domain.html">azurerm_eventgrid_domain</a>
                 </li>
 
+                <li<%= sidebar_current("docs-azurerm-resource-messaging-eventgrid-event-subscription") %>>
```

Comment:
I think this is an error? 🤔

Reply:
this appears to be otherwise missing from the sidebar (and shipped in #2967) - so this seems ok to me?

Reply:
Yea, this didn't make it in #2967 so I'm just adding it here